loader/cunit/cu_shp2pgsql.c

```
@@ -62,16 +69,124 @@ int clean_shp2pgsql_suite(void)
 	return 0;
 }
 
+static SHPLOADERCONFIG *
+make_topogeometry_config(void)
+{
+	SHPLOADERCONFIG *config = (SHPLOADERCONFIG*)calloc(1, sizeof(SHPLOADERCONFIG));
+	set_loader_config_defaults(config);
+	config->topology = strdup("city_topo");
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)